src/App.js

```diff
@@ -1,51 +1,23 @@
+import { Routes, Route } from 'react-router-dom';
+import About from './pages/About';
+import Faq from './pages/Faq';
+import Home from './pages/Home';
```

Comment:
These can be lazy-loaded so we decrease the final bundle size. Make sure you wrap the route components in suspense and show some sort of a loading indicator. Like so:


```suggestion
import React, { Suspense } from 'react';
const About = React.lazy(() => import('./pages/About'));
const Faq = React.lazy(() => import('./pages/Faq'));
const Home = React.lazy(() => import('./pages/Home'));
```

And then:
```
...
<Route
	  path="about"
	  element={
	    <React.Suspense fallback={<>...</>}>
	      <About />
	    </React.Suspense>
	  }
/>
...
```